script/code_style.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+echo "Running SwiftFormat..."
```

Comment:
Can we have this implemented in Ruby to be consistent? Otherwise we'll end up with automation logic written in Swift, Ruby, Bash, which makes the setup harder to reason about.

Reply:
I tried to implement it in the Rakefile, but when invoking `swiftlint`, I wasn't able to capture the error without terminating the Ruby process. I tried with `system` , backticks, `Open3.capture` etc... No luck, as soon as `swiftlint` errors the ruby process would stop. Any ideas?